connectors/filtering/validation.py

```diff
@@ -103,11 +103,17 @@ def __str__(self):
         return f"(ids: {self.ids}, messages: {self.messages})"
 
 
+class InvalidFilteringValidationStateError(ValueError):
+    pass
+
+
 class FilteringValidationState(Enum):
     VALID = 1
     INVALID = 2
     EDITED = 3
 
+    STATES = [VALID, INVALID, EDITED]
```

Comment:
nit, make it immutable:

```
STATES = VALID, INVALID, EDITED
```

Reply:
you can also use:

```
STATES = tupe(item.value for item in FilteringValidationState)
```

(I am not entirely sure why you need `STATES` though)

Reply:
Removed it by taking your suggestion further down into consideration 👍